docs/userguides/contracts.md

```diff
@@ -188,20 +190,33 @@ def set_number(num: uint256):
     assert msg.sender == self.owner, "!authorized"
     self.prevNumber = self.myNumber
     self.myNumber = num
+
+@external
+@payable
+def withdraw():
+    self.wdAmount = msg.value
 ```
 
-Notice the contract has both an external pure method and an external method that modifies state.
+Notice the contract has an external pure method, an external method that modifies state, and an external payable method that also modifies state using the given `msg.value`.
 In EVM languages, methods that modify state require a transaction to execute because they cost money.
 Modifying the storage of a contract requires gas and thus requires a sender with enough funding.
+Methods that accept value are `payable` (e.g. `msg.value` in Vyper); provide additional value (e.g. Ether) to these methods.
 Contract calls, on the other hand, are read-operations and do not cost anything.
+Calls are never payable.
 Thus, calls do not require specifying a `sender=` in Ape.
 
 At the RPC level, Ethereum calls are performed using the `eth_call` RPC and transactions are performed using the `eth_sendTransaction` or `eth_sendRawTransaction` RPCs.
 
+The following sub-sections show how, using Ape, we can invoke or call the methods defined above.
+
 ### Transactions
 
 The following example demonstrates invoking a contract's method in Ape as a transaction.
-However, take note that there is a [separate guide](./transactions.html) which fully covers transactions in Ape.
+Remember: transactions cost money, whether they are payable or not.
+Payable transactions cost more money, because the contract-logic accepts additional value (e.g. Ether).
```

Comment:
```suggestion
Payable transactions cost more money, because the contract-logic requires additional value (e.g. Ether) to be forwarded with the call.
```